test/command/domain-set-contacts-test.php

```diff
@@ -47,7 +47,7 @@ public function test_command_instance_success(): void {
 						Command\Command_Interface::KEY_CONTACT_DISCLOSURE => Entity\Contact_Disclosure::NONE,
 					],
 				],
-				Command\Command_Interface::KEY_TRANSFERLOCK_OPT_OUT => true,
+				Command\Command_Interface::KEY_TRANSFERLOCK_OPT_OUT => false,
```

Comment:
I think this was changed by mistake when the `KEY_TRANSFERLOCK_OPT_OUT` constant was renamed.

Reply:
This was fixed in #183, so this change was removed from this PR.